package.json

```diff
@@ -47,7 +47,7 @@
     "lint:scss": "stylelint {demosplan/DemosPlanCoreBundle/Resources/client/scss,projects}/**/*.scss"
   },
   "dependencies": {
-    "@demos-europe/demosplan-ui": "^0.3.38",
+    "@demos-europe/demosplan-ui": "^0.3.40",
```

Comment:
to prevent overrides later, i suggest we update to 0.3.41 (which was released today) if possible